### DISCUSSION TITLE
Zoomed Menu

### DISCUSSION THREAD

**Discussion by qlock99** [Polprzewodnikowy/N64FlashcartMenu#69] [General]
The menu for me appears zoomed in across multiple TVs. It navigates fine and when roms are booted they appear fine. I am wondering if I am missing something in the settings. I tried messing with the config file for the menu and settings in the sc64depeloyer with no success. Any ideas?
![PXL_20231120_023850647](https://github.com/Polprzewodnikowy/N64FlashcartMenu/assets/151485998/f53627b7-1dec-4d55-9fa2-2bbd8fa18a48)

**Comment by networkfusion** [comment]
This is a problem with overscan. hopefully fixed in future with current progress towards it in libdragon.
